data/en/test_calculated_summary.json

```diff
@@ -59,22 +59,22 @@
                         "type": "General",
                         "answers": [{
                                 "id": "second-number-answer",
-                                "label": "Second answer in currency total",
+                                "label": "Second answer in currency label",
                                 "mandatory": true,
                                 "type": "Currency",
                                 "currency": "GBP",
                                 "decimal_places": 2
                             },
                             {
                                 "id": "second-number-answer-unit-total",
-                                "label": "Second answer label in unit total",
+                                "label": "Second answer label in unit label",
```

Comment:
Petty but the label wording is now confusing. It was probably clearer before. This is the label of the second answer used in the Unit Total. Few confusing labels like this.

Reply:
That's a point fair.  It's got the word label in the title so I don't need to change it again at the end